debezium-testing/debezium-testing-testcontainers/src/main/java/io/debezium/testing/testcontainers/ConnectorConfiguration.java

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.testing.testcontainers;
+
+import java.util.List;
+
+import org.testcontainers.containers.JdbcDatabaseContainer;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+
+/**
+ * Class that represents the config element of the configuration document.
+ */
+public class ConnectorConfiguration {
+
+    private final ObjectMapper mapper = new ObjectMapper();
+    private final ObjectNode configNode;
+
+    protected ConnectorConfiguration() {
+        this.configNode = this.mapper.createObjectNode();
+        this.configNode.put("tasks.max", 1);
+    }
+
+    public static ConnectorConfiguration create() {
+        return new ConnectorConfiguration();
+    }
+
+    static ConnectorConfiguration from(JsonNode configNode) {
```

Comment:
Just to confirm, is this way how to configure MongoDB connector?

Reply:
You could; but more realistically, you'd start with an empty `ConnectorConfiguration` and populate it with `with(key, value)` calls.